lib/calculate-modulo-256-via-x86-assembly.yml

```diff
@@ -4,7 +4,7 @@ rule:
     authors:
       - moritz.raabe@mandiant.com
     lib: true
-    scope: instruction
+    scope: basic block
```

Comment:
Not a FP, but I think the `instruction` scope captures better what we're trying to get at here. In general, the basic block scope is too broad and results in more FPs. So, I'd vote to leave this one as is.

```asm
.text:0000000140010CC5 B8 FF 00 00 00    mov     eax, 0FFh
.text:0000000140010CCA 66 23 C8          and     cx, ax
```